engine/time/src/main/java/io/deephaven/time/calendar/BusinessCalendarXMLParser.java

```diff
@@ -262,7 +262,7 @@ private static TimeRange<LocalTime>[] parseBusinessRanges(final List<Element> bu
 
             final LocalTime open = DateTimeUtils.parseLocalTime(openTxt);
             final LocalTime close = DateTimeUtils.parseLocalTime(closeTxt);
-            rst[i] = new TimeRange<>(open, close, true);
```

Comment:
Wasn't this set to `true` so that "23:59:59.999999999" would include the last nanosecond in the day?

Right now, it looks like we _can't_ represent a full day?

Reply:
I seem to recall some discussion involving DHE? Maybe Japan business day, or some other international concerns?

Reply:
You're right that there may have been a UTC discussion.  I'll look at that case.

Reply:
The fix in there allows UTC to be supported correctly with full 24 hour days.  It also gives users control over how to count the close times.